tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -183,6 +183,18 @@ declare var cfg:ISnippetConfig;
 
             return envelope;
         }
+
+
+        let cdnEndpoints = [
```

Comment:
This is ok for initial testing, but eventually we will want to abstract this out (to reduce the size) to just be the domains, so then we would just replace the previous domain with the new one...

This is also required so we 't have to list every format (eg. scripts/b/ai.2.gbl.min.js etc).